contracts/acl/ACL.sol

```diff
@@ -145,6 +149,21 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermission(_entity, _app, _role, NO_PERMISSION);
     }
 
+    /**
+    * @dev Revokes all permissions for a specified role if allowed. This requires `msg.sender` to be the the permission manager
+    * @notice Revoke from all the ability to perform actions requiring `_role` on `_app`
+    * @param _app Address of the app in which the role will be revoked
+    * @param _role Identifier for the group of actions in app being revoked
+    */
+    function revokeAll(address _app, bytes32 _role)
+        external
+        onlyPermissionManager(_app, _role)
+    {
+        uint256 newRoleEra = roleEras[roleHash(_app, _role)] + 1;
```

Comment:
Sure, although it seems really unlikely that this function gets called `2^256` times for the same app and role, I guess you mean instead of the next line, right?

Reply:
I added SafeMath and will remove the require on the next line :)